.github/ISSUE_TEMPLATE/new-repo.md

```diff
@@ -21,18 +21,18 @@ Sponsoring WG:
 
 ## Actions to fulfill
 
-This area is used for the TOC to track the [repo creation
-process](/mechanics/CREATING-A-SANDBOX-REPO.md)
-
+This area is used for the TOC to track the [repo creation process](/mechanics/CREATING-A-SANDBOX-REPO.md).
+The _requestor_ should perform the steps listed below and cross out the checkmarks when done.
```

Comment:
It is a mix of the requestor and the sponsoring WG lead driving the steps.

Reply:
ok, let me try to clarify this.